flashinfer/cute_dsl/blockscaled_gemm.py

```diff
@@ -2702,3 +2702,91 @@ def dtype(cutlass_dtype):
         )
 
         return c_tensor_gpu
+
+
+def _dtype_to_cutlass_dtype(dtype: str) -> cutlass.dtype:
```

Comment:
Probably we could consider moving the helper func to `flashinfer/flashinfer/utils.py`